spin-up-hub/src/components/ContentListing.vue

```diff
@@ -24,13 +24,13 @@ export default {
             if (this.searchTerm) {
                 let updatedQuery = this.searchTerm
                     .split(" ")
-                    .map(word => word + '^2 ' + word + '* ' + word + '~2')
+                    .map(word => word + '^2 ' + word + '* ')
                     .join(' ');
                 let result = this.searchIndex.search(updatedQuery)
                 //get only confident results
                 let matches = []
                 result.map(k => {
-                    if (k.score < 0.5) {
+                    if (k.score < 1) {
```

Comment:
This is the right place but I would go higher with all the boosts we have, in my experiments something around 5 was good.